docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>Should we add this to examples instead as a deep-dive?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>This will replaces our old developer guide, so I think it belongs in the core documentation</p>